### ISSUE TITLE
Break repo apart into smaller, independent packages

### ISSUE THREAD

**Issue by gabeweaver** [spartansystems/booster-kit-react#22]
Using [Lerna](https://github.com/lerna/lerna), we can create a number of standalone packages that can be used on various projects. This will keep the core booster kit client lean, but add a clean interface for installing other managed dependencies designed to work seamlessly with the booster kit. 

Package roadmap:
- [ ] Webpack
- [ ] Feature Toggles
- [ ] Graphql
- [ ] Component Library
- [ ] Client (Server, Client, Universal)
- [ ] Docs
- [ ] Flow

**Comment by bbrock25** [comment]
@gabeweaver can you make this the top priority for your contributions here? It would be really great to write up a small blog post on learna and our usage here to accompany the PR.

**Comment by gabeweaver** [comment]
@bbrock25 yep...i'll start on this next. I wanted to get webpack updated and yarn installed first. Shouldn't take too long!

**Comment by bbrock25** [comment]
rawrsome, thanks man!

**Comment by gabeweaver** [comment]
This has kind of shifted in priority a bit. As of right now, the most common sense packages to extract are:
* Atom Library
* A collection of HOCs that are applicable regardless of where data is sourced from - `withForm`, `connectState`, etc. 

Leaving this open until a more clearly defined strategy emerges. I don't want to extract stuff out into packages unless we want to publish it to NPM (which we should with some of this stuff), but i dont think any of it is ready yet.
